tests/output/ConsoleInteractionTest.py

```diff
@@ -712,9 +712,7 @@ def test_show_bear_with_content(self):
                              '  Needed Settings:\n'
                              '   * setting1: Required Setting.\n\n'
                              '  Optional Settings:\n'
-                             '   * setting2: Optional Setting. ('
-                             "Optional, defaults to 'None'."
-                             ')\n\n'
+                             '   * setting2: Optional Setting.\n\n'
```

Comment:
The issue is more about separating the two. Changing the UI/output is a different thing. Feel free to add it as a separate commit :) (FWIW, I agree, we don't need the "Optional" qualifier for every setting.)

The reason I filed the issue is, I wanted to get the default value for each setting to display as a column in bear-docs.

Anyway, this is how I'd like to see it:

`self.optional_params["use_spaces"] = ("setting description", <class 'str'>, True)`

So we can just remove the fourth "Optional" string. But remember to reconstruct the original help text like this:

`   * tab_width: Number of spaces per indentation level. (Optional, defaults to '4'.)"`